app/lib/data_exporter.rb

```diff
@@ -0,0 +1,42 @@
+require "yaml"
+
+class DataExporter
+  def self.call
+    new.call
+  end
+
+  def call
+    Rails.logger.info licence_data.each(&:deep_stringify_keys!).to_yaml
+  end
+
+  def licence_data
+    licences = Licence.all
+    licences.map do |licence|
+      {
+        licence_identifier: licence.gds_id,
+        locations: locations(licence),
+        industry_sectors: level_two_sectors(licence),
+      }
+    end
+  end
+
+private
+
+  def locations(licence)
+    locations = []
+    locations << "england" if licence.da_england
+    locations << "wales" if licence.da_wales
+    locations << "scotland" if licence.da_scotland
+    locations << "northern-ireland" if licence.da_northern_ireland
+
+    locations
+  end
+
+  def level_two_sectors(licence)
+    sectors = Sector.in(id: LicenceLink.where(licence_id: licence.id).pluck(:sector_id))
+    sectors.map do |sector|
+      parent_sectors = sector.parents.to_a
+      parent_sectors.first.name.parameterize if parent_sectors.first&.layer == 2
```

Comment:
We should probably record and print out when / if this happens to see if our assumption is correct. Currently we'll just get `nil` values in the array if this is true.

Reply:
Hmm that's true. I think I might have fallen foul of this when I was setting up the tests. I'll remove the line and test it against integration again.

Reply:
Actually, can I clarify? 

Do you think the guard clause `if parent_sectors.first&.layer == 2` isn't needed at all, or just the safe navigation operator?

Reply:
Ok. I'll have a little play around in integration out of curiosity,  however I don't think the guard clause hurts in general

Reply:
Ok, so there aren't any LicenceLinks to layer 1 or 2 sectors on integration: https://deploy.integration.publishing.service.gov.uk/job/run-rake-task/305278/console

Some of the licences are tagged to an astonishing amount of layer 3 sectors though. I think I saw 89 for one of them!

Reply:
Thanks for checking, wow... 89 sectos 😱

If we've confirmed there are none, then we can remove the guard clause completely as the data isn't going to change 👍

Reply:
Oh, I should mention, that I've replaced `map` with `filter_map` as that's the equivalent of doing `map` followed by `compact` to remove nils.

Reply:
Okay, sure (concerning guard clause) 👍  

> Oh, I should mention, that I've replaced map with filter_map as that's the equivalent of doing map followed by compact to remove nils.

Ah nice, I didn't know `filter_map` existed!